crypto/x509/x509_set.c

```diff
@@ -192,8 +192,8 @@ int X509_get_signature_info(X509 *x, int *mdnid, int *pknid, int *secbits,
     return X509_SIG_INFO_get(&x->siginf, mdnid, pknid, secbits, flags);
 }
 
-static void x509_sig_info_init(X509_SIG_INFO *siginf, const X509_ALGOR *alg,
-                               const ASN1_STRING *sig)
+static int x509_sig_info_init(X509_SIG_INFO *siginf, const X509_ALGOR *alg,
```

Comment:
Since you make it return something, please at least document the return value. A short description would be nice.

Reply:
All right, done.